src/core/models/components.ts

```diff
@@ -223,6 +224,17 @@ const components = createModel({
           parent: payload.parentName,
           rootParentType: payload.rootParentType || payload.type,
         }
+        if (ComponentWithRefs.includes(payload.type)) {
+          const ref = `${id.replace('-', '_')}_ref`.toLowerCase()
```

Comment:
use camel case

Reply:
This is how it is done right now: input_l9k07h0g8nf2o_ref
After CamelCase it will look like this: inputl9k07h0g8nf2oRef 
I feel it is more clear following the snake case.

Reply:
OR I can put ref first like this:
RefInputl9k07h0g8nf2o

Reply:
this will translate to variable name so you can make it refInput_l9k07h0g8nf2o
also reduce the length of random string

Reply:
done!